### PR TITLE
Update readmes to match common design (Geometry category) 3/3

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: geometry/list-transformations-by-suitability/README.md [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
 
 ## How to use the sample
 
-The list displays all suitable tranformations between the graphic and the map. Once the transform button is clicked a new red graphic will appear showing where the original graphic would be placed if transformation was applied.
+Select a transformation from the list and click 'Transform' to see the result of projecting the point from EPSG:27700 to EPSG:3857 using that transformation. The result is shown as a red cross; you can visually compare the original blue point with the projected red cross. 
 
-Order by extent suitability, if checked, will find suitable transformations within the map's visible area. If not checked, will find suitable transformations using the whole map.
+Select 'Consider current extent' to limit the transformations that are appropriate for the current extent.
```
Comment (Rachael-E) [inline geometry/list-transformations-by-suitability/README.md]:
```suggestion
Select 'Order by extent suitability' to limit the transformations that are appropriate for the current extent.
```

File: geometry/list-transformations-by-suitability/README.md [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
+1. Pass the input and output spatial references to `TransformationCatalog.getTransformationsBySuitability()` for transformations based on the map's spatial reference OR additionally provide an extent argument to only return transformations suitable to the extent. This returns a list of ranked transformations.
+2. Use one of the `DatumTransformation` objects returned to project the input geometry to the output spatial reference.
 
 ## Relevant API
 
-* ArcGISMap
-* Basemap
 * DatumTransformation
+* GeographicTransformation
+* GeographicTransformationStep
```
Comment (Rachael-E) [inline geometry/list-transformations-by-suitability/README.md]: looks like we don't have these classes/methods in our API/sample (I can't see them in our sample)

File: geometry/spatial-operations/README.md [hunk 3]
```diff
@@ -2,20 +2,20 @@
```
Comment (Rachael-E) [inline geometry/spatial-operations/README.md:2]: For the title, it should be sentence case

File: geometry/spatial-operations/README.md [hunk 4]
```diff
@@ -2,20 +2,20 @@
...
-![](SpatialOperations.png)
+![Image of spatial operations](SpatialOperations.png)
+
+## Use case
+
+Determining the spatial relationships between points, lines or polygons is a fundamental concept in GIS. For example, a data analyst may need to find the intersection between areas with a high concentration of medical emergency situations and the service area of a hospital.
 
 ## How to use the sample
 
 The sample provides a drop down on the top, where you can select a geometry operation. When you choose a geometry operation, the application performs this operation between the overlapping polygons and applies the result to the geometries.
```
Comment (Rachael-E) [inline geometry/spatial-operations/README.md:13]: looks like there's an unneeded space at the start of this sentence

**Review by Rachael-E — COMMENTED** [review]
just some typos/format and update of UI names for this one, thanks @JonLavi !

File: geometry/list-transformations-by-suitability/README.md (hunk 2 above)
Comment (JonLavi) [reply geometry/list-transformations-by-suitability/README.md]: Good catch, thanks!

File: geometry/spatial-operations/README.md (hunk 3 above)
Comment (JonLavi) [reply geometry/spatial-operations/README.md:2]: Thanks, updated to sentence case

File: geometry/spatial-operations/README.md (hunk 4 above)
Comment (JonLavi) [reply geometry/spatial-operations/README.md:13]: I can't see it! Maybe it's the github diff acting up?

**Comment by JonLavi** [comment]
Thanks @Rachael-E , I've updated with your comments. Could you have a second look please?

File: geometry/spatial-operations/README.md (hunk 4 above)
Comment (Rachael-E) [reply geometry/spatial-operations/README.md:13]: I think it's the diff yes! When it shows a plus for new change it auto bumps the text over and makes it look like there's a space I've learned. Sorry!

**Review by Rachael-E — APPROVED** [review]

**Pull request merged**